### PR TITLE
Handled Calendar PR change requests 

### PR DESCRIPTION
**Pull request by mtchang25**
calendar changes

- Cleaned up css. 
- Moved Calendar logic to pages. 
- Deleted CalendarStyles.js file

TODO: Implement Nextjs get static props to dynamically render data once BE team completes data implementation and pushes to repo.

### CODE REVIEW

File: components/Calendar/subcomponents/TooltipContent.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+export default function AppointmentContent (e) {
```
Comment (cmjosephs) [inline components/Calendar/subcomponents/TooltipContent.js]: e is not descriptive as a prop name. Single name parameters and variables should only be used in a few instances, such as for loops and synthetic react events. Rename to props or something more descriptive

File: pages/app/calendar.js [hunk 2]
```diff
@@ -1,7 +1,69 @@
...
+
+
+export default function Calendar () {
+//import user state (mentor/mentee)
+
+  const [appointmentData, setAppointmentData] = useState([]);
+
+
+    useEffect(() => {
+      setAppointmentData(data);
```
Comment (cmjosephs) [inline pages/app/calendar.js:36]: Later refactor to use SSG or SSR when fetching data. should mostly not need to use useEffect hook in /pages components

**Review by cmjosephs — CHANGES_REQUESTED** [review]
Please read 2 comments. Afterwards go ahead and merge. Good work!

File: components/Calendar/subcomponents/TooltipContent.js (hunk 1 above)
Comment (mtchang25) [reply components/Calendar/subcomponents/TooltipContent.js]: On it!

File: pages/app/calendar.js (hunk 2 above)
Comment (gunpowder66) [reply pages/app/calendar.js:36]: yea we're just waiting for api calls, this was temporary

**Comment by mtchang25** [comment]
Looks good to me! Fixed merge conflicts in package json!

**Pull request merged**